components/group/GroupRelatedInvitations.js

```diff
@@ -28,6 +28,7 @@ const GroupRelatedInvitations = ({ group, accessToken }) => {
       : {
           prefix: groupId.includes(submissionName) ? `${groupId}/.*` : `${groupId}/-/.*`,
           expired: true,
+          trash: true,
```

Comment:
@carlosmondra said you can not get all the expired and trashed invitations in the same call, is that right?